validation/src/test/scala/hmda/validation/engine/lar/quality/LarQualityEngineSpec.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.engine.lar.quality
+
+import hmda.parser.fi.lar.LarGenerators
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class LarQualityEngineSpec
+    extends PropSpec
+    with PropertyChecks
+    with MustMatchers
+    with LarGenerators
+    with LarQualityEngine {
+
+  property("A LAR must pass quality checks") {
+    forAll(larGen) { lar =>
+      whenever(lar.id == 2) {
+        checkQuality(lar).isSuccess mustBe true
+      }
+    }
+  }
```

Comment:
I'm a little skeptical of this spec, but no more than the ones it's based on. I'm not convinced that all our generated LARs will pass quality checks. That said, if it's currently passing, then I wouldn't let it stand in the way of merging. Probably.